app/parse.py

```diff
@@ -8,9 +18,102 @@ class Quote:
     tags: list[str]
 
 
+@dataclass
+class Author:
+    biography: str
+
+
+def parse_single_quote(page_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=page_soup.select_one(".text").text,
+        author=page_soup.select_one(".author").text,
+        tags=[tag.text for tag in page_soup.select(".tag")]
+    )
+
+
+def parse_single_author(page_soup: BeautifulSoup) -> Author:
+    return Author(
+        biography=page_soup.select_one(
+            ".author-details"
+        ).text.replace("\n", " "),
+    )
+
+
+def write_quotes_in_file(quotes: [Quote], output_csv_path: str) -> None:
+    quote_fields = [field.name for field in fields(Quote)]
+    with open(output_csv_path, "w", newline="", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        writer.writerow(quote_fields)
+        writer.writerows([astuple(quote) for quote in quotes])
+
+
+def write_authors_in_file(authors: [Author]) -> None:
+    author_fields = [field.name for field in fields(Author)]
+    with open("authors.csv", "w", newline="", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        writer.writerow(author_fields)
+        writer.writerows([astuple(author) for author in authors])
+
+
+def get_authors_page_soup(
+        page_soup: list[BeautifulSoup]
+) -> list[BeautifulSoup]:
+    list_authors = []
```

Comment:
list_authors -> authors
Why you add prefix list. We know it already)

Reply:
fix